configure.ac

```diff
@@ -69,4 +69,35 @@ AC_CONFIG_FILES([Makefile
                  tests/Makefile
                  ublksrv.pc])
 
+#
+# Check if g++ supports -fcoroutines -std=c++20
+#
+AC_LANG_PUSH([C++])
+CXXFLAGS_save=$CXXFLAGS
+CXXFLAGS="$CXXFLAGS -fcoroutines -std=c++20"
```

Comment:
I'd rather not add it in this way, and libublksrv doesn't need c++20 coroutine, and only the target code requires it.